metadata-ingestion/src/datahub/ingestion/api/registry.py

```diff
@@ -184,6 +184,12 @@ def get(self, key: str) -> Type[T]:
             # If it's not an exception, then it's a registered type.
             return tp
 
+    def get_optional(self, key: str) -> Optional[Type[T]]:
+        try:
+            return self.get(key)
+        except Exception:
+            return None
```

Comment:
Isn't there a nicer way to get None instead of a silently caught exception?
Do we care about the exception or is it fine to drop silently?

Reply:
in this case, we don't care about the exception